src/managers/KeybindManager.cpp

```diff
@@ -579,6 +580,19 @@ void CKeybindManager::toggleActiveFloating(std::string args) {
     g_pLayoutManager->getCurrentLayout()->changeWindowFloatingMode(PWINDOW);
 }
 
+void CKeybindManager::centerWindow(std::string args) {
+    CWindow* PWINDOW = nullptr;
+
+    PWINDOW = g_pCompositor->m_pLastWindow;
```

Comment:
why is this two lines? also, PWINDOW can be made const. That's what ALLCAPS means.